includes/abstracts/abstract.llms.database.query.php

```diff
@@ -163,23 +164,33 @@ protected function get_default_args() {
 				'id' => 'ASC',
 			),
 			'suppress_filters' => false,
+			'no_found_rows'    => false,
 		);
 
 		if ( $this->get( 'suppress_filters' ) ) {
 			return $args;
 		}
 
+		/**
+		 * Filters the query default args
+		 *
+		 * @since unknown
```

Comment:
Safe to assume that the introduction of the filter came with the introduction of the method since there haven't been any updates to the function since its introduction!

```suggestion
		 * @since 3.8.0
```